inception/inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/LayerStatistics.java

```diff
@@ -0,0 +1,157 @@
+            double aStandardDeviation, double aMaximumPerSentence, double aMinimumPerSentence,
+            double aMeanPerSentence, double aMedianPerSentence,
+            double aStandardDeviationPerSentence, long aNoOfDocuments)
+    {
+        total = aTotal;
+        maximum = aMaximum;
+        minimum = aMinimum;
+        mean = aMean;
+        median = aMedian;
+        standardDeviation = aStandardDeviation;
+
+        maximumPerSentence = aMaximumPerSentence;
+        minimumPerSentence = aMinimumPerSentence;
+        meanPerSentence = aMeanPerSentence;
+        medianPerSentence = aMedianPerSentence;
+        standardDeviationPerSentence = aStandardDeviationPerSentence;
+
+        noOfDocuments = aNoOfDocuments;
+    }
+
+    public boolean isLayerTrivial()
+    {
+        if (maximum == 0L) {
+            return true;
+        }
+        return false;
+    }
+
+    public long getTotal()
+    {
+        return total;
+    }
+
+    public long getMaximum()
+    {
+        return maximum;
+    }
+
+    public long getMinimum()
+    {
+        return minimum;
+    }
+
+    public double getMean()
+    {
+        return mean;
+    }
+
+    public double getMedian()
+    {
+        return median;
+    }
+
+    public double getStandardDeviation()
+    {
+        return standardDeviation;
+    }
+
+    public double getMaximumPerSentence()
+    {
+        return maximumPerSentence;
+    }
+
+    public double getMinimumPerSentence()
+    {
+        return minimumPerSentence;
+    }
+
+    public double getMeanPerSentence()
+    {
+        return meanPerSentence;
+    }
+
+    public double getMedianPerSentence()
+    {
+        return medianPerSentence;
+    }
+
+    public double getStandardDeviationPerSentence()
+    {
+        return standardDeviationPerSentence;
+    }
+
+    public long getNoOfDocuments()
+    {
+        return noOfDocuments;
+    }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (o == this) {
+            return true;
+        }
+
+        if (!(o instanceof LayerStatistics)) {
+            return false;
+        }
+
+        LayerStatistics ls = (LayerStatistics) o;
+
+        return ((ls.getTotal() == getTotal()) && (ls.getMaximum() == getMaximum())
+                && (ls.getMinimum() == getMinimum()) && (ls.getMedian() == getMedian())
+                && (ls.getMean() == getMean())
+                && (ls.getStandardDeviation() == getStandardDeviation())
+                && (ls.getNoOfDocuments() == getNoOfDocuments())
+                && (ls.getMaximumPerSentence() == getMaximumPerSentence())
+                && (ls.getMinimumPerSentence() == getMinimumPerSentence())
+                && (ls.getMedianPerSentence() == getMedianPerSentence())
+                && (ls.getMeanPerSentence() == getMeanPerSentence())
+                && (ls.getStandardDeviationPerSentence() == getStandardDeviationPerSentence()));
+    }
```

Comment:
I would save layer and feature name also in this to make it simpler to find out for what these stats are. Maybe you do not need it, then you can ignore my comment

Reply:
LayerStatistics is just a helper class. Everything in it is only accessed through StatisticsResult where layer and feature are saved.